main.cpp

```diff
@@ -326,6 +578,29 @@ Thing 10) Elevator
     2) Open and close
     3) Remember floor chosen by user
  */
+struct Elevator
+{
+
+    //Make new Cabin
+    Cabin lobbyElevatorCabinOne;
+    //Make new Door
+    Door lobbyElevatorDoor_one, lobbyElevatorDoor_two;
+    //Make new Panel
+    Panel lobbyElevatorPanel;
+    //Make new Control
+    Control lobbyElevatorControl;
+    //Make new Motor
+    Motor lobbyElevatorMotor;
```

Comment:
You do not need to include the word lobby elevator here.
Lobby elevator refers to a specific instance of an elevator.
You could have a separate elevator, in the back of the building. Perhaps it is the service elevator. 
If you're trying to access that elevators cabin variable, it's gonna look funny:
```
Elevator serviceElevator;
serviceElevator.lobbyElevatorCabinOne.someMemberFunction();
```
That just reads and looks silly. 

Name things what they are, don't include any extra adjectives.
```
Cabin cabin;
Door leftDoor, rightDoor;
Panel panel;
Control control;
Motor motor;
```

When you declare instances of the elevator` **that's** when you specify that it is a lobby elevator, a service elevator, a garage elevator, etc.
```
Elevator lobbyElevator, serviceElevator, garageElevator;
```

- [ ] revise

Reply:
Ah, ok. Yes I understand this now